<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>933</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(236, 236, 236);
	color: rgb(36, 36, 36);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="InputBox">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>670</y>
      <width>461</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	font: 11pt;
	border: 0px;
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'微软雅黑';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>650</y>
      <width>501</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 7px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="EnableEnterSend">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>600</y>
      <width>131</width>
      <height>38</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
	font: 10pt;
}
QCheckBox::hover {
	color: rgb(40, 176, 104);
}</string>
    </property>
    <property name="text">
     <string>回车键发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendMessage">
    <property name="geometry">
     <rect>
      <x>403</x>
      <y>596</y>
      <width>105</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 10.5pt;
	color: rgb(255, 255, 255);
	background-color: rgb(104, 224, 104);
	border: 1px solid;
	border-color: rgb(249, 249, 249);
	border-radius: 6px;
}
QPushButton::hover {
	background-color: rgb(109, 236, 109);
}

QPushButton::disabled {
	background-color: rgb(169, 224, 166);
}</string>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="TheOtherChatting">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>80</y>
      <width>241</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser {
	font: 11pt;
	border: 0px;
}</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="MyChatting">
    <property name="geometry">
     <rect>
      <x>282</x>
      <y>80</y>
      <width>241</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser {
	font: 11pt;
	border: 0px;
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>InputBox</zorder>
   <zorder>EnableEnterSend</zorder>
   <zorder>SendMessage</zorder>
   <zorder>MyChatting</zorder>
   <zorder>TheOtherChatting</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>568</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="dialog">
    <property name="title">
     <string>会话</string>
    </property>
    <addaction name="new_dialog"/>
    <addaction name="close_dialog"/>
    <addaction name="close_every_dialog"/>
   </widget>
   <widget class="QMenu" name="myfriend">
    <property name="title">
     <string>好友</string>
    </property>
    <addaction name="add_friend"/>
    <addaction name="friend_list"/>
   </widget>
   <widget class="QMenu" name="option">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="exit_login"/>
    <addaction name="add_new_id"/>
    <addaction name="settings"/>
    <addaction name="exit"/>
   </widget>
   <addaction name="dialog"/>
   <addaction name="myfriend"/>
   <addaction name="option"/>
  </widget>
  <action name="new_dialog">
   <property name="text">
    <string>新建会话</string>
   </property>
  </action>
  <action name="close_dialog">
   <property name="text">
    <string>关闭会话</string>
   </property>
   <property name="iconText">
    <string>关闭会话</string>
   </property>
  </action>
  <action name="close_all_dialog">
   <property name="text">
    <string>关闭所有会话</string>
   </property>
  </action>
  <action name="add_friend">
   <property name="text">
    <string>添加好友</string>
   </property>
  </action>
  <action name="friend_list">
   <property name="text">
    <string>好友列表</string>
   </property>
  </action>
  <action name="settings">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="exit_login">
   <property name="text">
    <string>退出登录</string>
   </property>
  </action>
  <action name="add_new_id">
   <property name="text">
    <string>添加新号码</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>退出应用(同时退出登录)</string>
   </property>
  </action>
  <action name="close_every_dialog">
   <property name="text">
    <string>关闭所有会话</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
